pkgs/development/python-modules/asyncssh/default.nix

```diff
@@ -23,17 +40,20 @@ buildPythonPackage rec {
     pyopenssl
   ];
 
-  checkInputs = [ openssl ];
+  checkInputs = [
+    openssh
+    openssl
+  ];
 
   # Disables windows specific test (specifically the GSSAPI wrapper for Windows)
   postPatch = ''
-    rm ./tests/sspi_stub.py
+    rm tests/sspi_stub.py
   '';
 
   meta = with stdenv.lib; {
     description = "Provides an asynchronous client and server implementation of the SSHv2 protocol on top of the Python asyncio framework";
-    homepage    = https://pypi.python.org/pypi/asyncssh;
-    license     = licenses.epl20;
+    homepage = https://pypi.python.org/pypi/asyncssh;
```

Comment:
```suggestion
    homepage = http://asyncssh.timeheart.net/;
```

Reply:
Redirects to https://asyncssh.readthedocs.io/en/latest/